bears/general/CPDBear.py

```diff
@@ -75,8 +75,8 @@ def run(self, language: str, minimum_tokens: int=10,
                      "--files", files,
                      "--format", "xml")
 
-        for option in options:
-            if options[option] is True:
+        for option, enable in options.items():
+            if enable is True:
```

Comment:
and yeah:
`arguments +=  [option for option, enable in options.items() if enable]`